QualityControl/test/test-index.js

```diff
@@ -167,6 +163,9 @@ suite('All Tests - QCG', { timeout: FRONT_END_TIMEOUT + BACK_END_TIMEOUT }, asyn
 
     suite('Controllers - Test Suite', async () => {
       suite('LayoutController test suite', async () => await layoutControllerTestSuite());
+      suite('UserController test suite', async () => {
+        // TODO - bring inline with current tests
```

Comment:
Do you plan to push this in this PR?
If not please remove this, it is dead code and it should be avoided